Earthfile

```diff
@@ -9,7 +9,7 @@ ARG LUET_VERSION=0.35.1
 # renovate: datasource=docker depName=aquasec/trivy
 ARG TRIVY_VERSION=0.50.1
 # renovate: datasource=github-releases depName=kairos-io/kairos-framework
-ARG KAIROS_FRAMEWORK_VERSION=v2.7.32
+ARG KAIROS_FRAMEWORK_VERSION=v2.7.33
```

Comment:
is this correct? shouldnt .11 have this framework version already??

Reply:
@Itxaka it does 

https://github.com/kairos-io/kairos/blob/release/v3.0.11/Earthfile#L12

strange 🧐

Reply:
fixed